cmd/root.go

```diff
@@ -83,9 +83,9 @@ func prerun(cmd *cobra.Command, args []string) {
 			BatchSize: 1,
 		})
 
-		username = os.Getenv("USER")
+		usr := os.Getenv("USER")
```

Comment:
nit: rather than assign it and cause the clobbering at all, you could just inline it:
```
analyticsClient.Enqueue(analytics.Identify{
    UserId: os.Getenv("USER"),
    Traits: analytics.NewTraits().Set("aws-okta-version", version),
})
```

Reply:
agreed that inline is better, I'm going to be lazy and not change it now though